### PR TITLE
Allow users to specify the data-trackable attribute for the teaser

### PR DESCRIPTION
**Pull request by bjfletcher**
This introduces another attribute to the teaser: data-trackable

There are data-trackable for various bits within the teaser (e.g. link type) however we need a data-trackable for the teaser as a whole. One example is to add details where the teaser is placed on the frontpage:

- `data-trackable="slice:hero-slice-1;slicePos:1;layout:standaloneimage;listPos:1"` (first top story)

- `data-trackable="slice:freeform-twocolumns;slicePos:5;layout:standaloneimage;listPos:14"` (last top story)

See https://github.com/Financial-Times/next-front-page/pull/2392 for the code there to add the above tracking. That PR would follow this PR (& release).

So, in short, this PR simply allows users of `x-teaser` to specify, optionally, a tracking value for the teaser.

### CODE REVIEW

File: components/x-teaser/storybook/top-story.js [hunk 1]
```diff
@@ -17,6 +17,7 @@ exports.knobs = [
 	'metaPrefixText',
 	'metaSuffixText',
 	'metaLink',
+	'dataTrackable',
```
Comment (taraojo) [inline components/x-teaser/storybook/top-story.js:20]: This only shows for the top story teaser in Storybook and not the other teasers, I'm not sure how important it is for it to show up for all of them. Was there a specific reason you only added to `top-story` @bjfletcher?

Comment (bjfletcher) [reply components/x-teaser/storybook/top-story.js:20]: No good reason (we're currently analysing only top stories) and I think that's a good idea to add to all the other types for the storybook. So I've just pushed a commit to add the knob to all the other teaser types.

**Review by taraojo — APPROVED** [review]
Looks good to me, thanks Ben

**Comment by taraojo** [comment]
I'll look at merging and releasing this tomorrow morning @bjfletcher as I'm a lot less familiar with that process
